self-hosted-toolbox

```diff
@@ -0,0 +1 @@
+source ~/.toolbox/libcheckout
```

Comment:
What is the purpose of this file?

Reply:
Same purpose as https://github.com/semaphoreci/toolbox/blob/master/toolbox. Basically, just sourcing all the available shell scripts